bin/dbatools-buildref-index.json

```
@@ -4394,6 +4438,10 @@
             "CU": "CU16",
             "Version": "15.0.4223",
             "KBList": "5011644"
-        }
+        },
+        {
+            "Version": "15.0.4236",
+            "KBList": "5014353"
+        },

```

thank you so much! a bunch of tests are failing and I think it's this last lil comma causing the problems

```suggestion
        }
```

---

lemme see if this works -- that's line 4445, removing a comma